src/api.ts

```diff
@@ -120,7 +126,7 @@ export interface StaticPlatformPlugin extends PlatformPlugin {
    *
    * @param {(foundAccessories: AccessoryPlugin[]) => void} callback
    */
-  accessories(callback: (foundAccessories: AccessoryPlugin[]) => void): void;
+  accessories(callback: (foundAccessories: Array<AccessoryPlugin & AccessoryConfig>) => void): void;
```

Comment:
The code assumes things like `name` and potentially `uuid_base` exist for a static platform, so declare it as such.